ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -270,24 +270,25 @@ gcp_flush(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session, WT_FILE_SYSTEM
     gcp_store *gcp = reinterpret_cast<gcp_store *>(storage_source);
     gcp_file_system *fs = get_gcp_file_system(file_system);
     WT_FILE_SYSTEM *wt_file_system = fs->wt_file_system;
+    std::string local_file_path = fs->home_dir + "/" + source;
 
     // std::filesystem::canonical will throw an exception if object does not exist so
     // check if the object exists.
-    if (!std::filesystem::exists(source)) {
+    if (!std::filesystem::exists(local_file_path)) {
         gcp->log->log_error_message(
-          "gcp_flush: Object: " + std::string(object) + " does not exist.");
+          "gcp_flush: Object: " + std::string(object) + " No such file.");
```

Comment:
(This goes for the Azure version of this code too.) This would read a little weird, right? Since now it'd say `/foo/bar No such file.`

Reply:
Yep that's better, cheers.